src/style.css

```diff
@@ -11,19 +11,40 @@ body {
 }
 
 .stars__star {
-  background-image: url("/images/star.svg");
+  background-image: url("/images/star-active.svg");
   background-position: center;
   background-repeat: no-repeat;
   width: 16px;
   height: 16px;
   margin-left: 4px;
 }
 
-.stars__star--active {
-  background-image: url("/images/star-active.svg");
+.stars--0 > .stars__star:nth-child(n+1) {
+  background-image: url("/images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 > .stars__star:nth-child(n+2) {
+  background-image: url("/images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--2 > .stars__star:nth-child(n+3) {
+  background-image: url("/images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--3 > .stars__star:nth-child(n+4) {
+  background-image: url("/images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--4 > .stars__star:nth-child(n+5) {
+  background-image: url("/images/star.svg");
   background-position: center;
   background-repeat: no-repeat;
-  width: 16px;
-  height: 16px;
-  margin-left: 4px;
 }
```

Comment:
that's good, but you can make it even better 😄 
it seems like you're doing a lot of very repetetive work with those styles, try to stack them inside one block with `,` operator

and you don't really need to change anything about background - it's already set in `.stars__star` class